docs/content/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yba.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Back up and restore YugabyteDB Anywhere
+headerTitle: Back up and restore YugabyteDB Anywhere
+description: Use a script to back up and restore YugabyteDB Anywhere.
+headcontent: Back up your YugabyteDB Anywhere installation
+linkTitle: Back up YugabyteDB Anywhere
+menu:
+  stable_yugabyte-platform:
+    identifier: back-up-restore-1-yba
+    parent: administer-yugabyte-platform
+    weight: 30
+type: docs
+---
+
+YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../back-up-restore-yba/" class="nav-link active">
+      <i class="fa-solid fa-cloud"></i>
+      YugabyteDB Anywhere
+    </a>
+  </li>
+  <li>
+    <a href="../back-up-restore-installer/" class="nav-link">
+      <i class="fa-solid fa-building"></i>
+      YBA Installer</a>
+  </li>
+  <li>
+    <a href="../back-up-restore-k8s/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+You can perform regularly scheduled backups of your YugabyteDB Anywhere installation to external storage. You can also perform ad hoc manual backups.
+
+Before you can back up to external storage, you need to [configure a storage location](../../back-up-restore-universes/configure-backup-storage/) for your backups.
```

Comment:
This seems to be necessary only for Automatic backups as manual are stored locally. So move it as a pre-req under automatic backups?